.github/workflows/prod-deploy-web.yml

```diff
@@ -35,9 +28,6 @@ jobs:
     secrets: inherit
 
   deploy_web_us:
```

Comment:
Run US/EU deployment in parallel. Let me know if there was a reason we were doing them sequentially.

Reply:
There is no good reason to do them separately.